init

```diff
@@ -50,15 +62,26 @@ modprobe dm-mod
 vgscan --mknodes --ignorelockingfailure >/dev/null 2>&1 || rescue_shell
 vgchange --sysinit --activate y >/dev/null 2>&1 || rescue_shell
 
+msg "Scanning for live media detection..."
+sfs="seiba/seiba.sfs"
+while [ "$live" = "1" ] && [ "$root" == "" ] ; do
+	list=$(ls /sys/class/block/ | grep ".*[0-9]$" | grep -v loop | grep -v ram | grep -v nbd | sed "s|^|/dev/|g")
+	for part in $list
+	do
+		sleep 0.1
+		if is_file_avaiable "$part" "${sfs}"
+		then
+			msg "Detected live media $part"
+			export root=$part
+		fi
+	done
+done
+
+
 if [ "$live" = "1" ]; then
 	msg "Mounting live media"
-	mount -n -r -t iso9660 LABEL="ataraxialinux" /newroot/cdrom || \
-	{
-		sleep 3
-		mount -n -r /dev/disk/by-label/ataraxialinux /newroot/cdrom || rescue_shell
-	}
- 
-	mount -n -r -o loop -t squashfs /newroot/cdrom/seiba/seiba.sfs /newroot/squash || rescue_shell
+	mount -t auto $root /newroot/cdrom || rescue_shell
+	mount -n -r -o loop -t squashfs /newroot/cdrom/${sfs} /newroot/squash || rescue_shell
```

Comment:
rufus, unetbootin or other shitty iso image writers never create iso9690 formatted partition. We detected live media partition. We can mount directly. (we dont need sleep because live detection has delay)

Reply:
sleep if failure

Reply:
or we can use while loop
```
while ! mount xxxxx
do
   sleep 0.1
done
```